### DISCUSSION TITLE
What are `cssFilename` and `cssChunkFilename` in Output?

### DISCUSSION THREAD

**Discussion by PierBJX** [webpack/webpack#15381] [Q&A] [answered]
Looking at the Output types, I noticed these two properties `cssFilename` and `cssChunkFilename`. I would like to know  what is the purpose of these two properties and how to use it? Indeed, I did not find the documentation on that.

**Comment by alexander-akait** [comment, marked as answer]
It is under construction and experiment with built-in CSS support https://webpack.js.org/configuration/experiments/#experimentscss
